docs/releases/v2020.2.rst

```diff
@@ -189,3 +190,13 @@ Known issues
   (`#94 <https://github.com/freifunk-gluon/gluon/issues/94>`_)
 
   Reducing the TX power in the Advanced Settings is recommended.
+
+* The MAC address of the WAN interface is modified even when Mesh-on-WAN is disabled
+  (`#496 <https://github.com/freifunk-gluon/gluon/issues/496>`_)
+
+  This may lead to issues in environments where a fixed MAC address is expected (like VMware when promiscuous mode is
+  disallowed).
+
+* Inconsistent respondd API (`#522 <https://github.com/freifunk-gluon/gluon/issues/522>`_)
+
+  The current API is inconsistent and will be replaced eventually. The old API will still be supported for a while.
```

Comment:
We're dragging this with us since years. I'm not sure whether or not we want to continue doing so.

Reply:
but simply leaving it out from release notes while still having it marked and not discussing it is not the way to go

Reply:
As long as we don't have a plan for a replacement it makes no sense to carry that remark.